app/controllers/spotlight/appearances_controller.rb

```diff
@@ -22,20 +27,34 @@ def edit
 
     protected
 
+    def update_masthead
+      @exhibit.masthead ||= Spotlight::Masthead.new
+      @exhibit.masthead.update(params.require(:exhibit).require(:masthead_attributes).permit(featured_image_params))
+      @exhibit.save
```

Comment:
This `masthead ||= Spotlight::Masthead.new` pattern shows up in a few places.  I feel like there is a better rails-y way to do this that is escaping me right now.